lib/helpers.js

```diff
@@ -13,21 +27,63 @@ export function locToRange(loc: Object): [[number, number], [number, number]] {
 
 export function processDeclaration(entries: Array<Object>, sourceFile: string): Array<Object> {
   const toReturn = []
+  
+  const openProjects = atom.project.getPaths()
+  let projectPath
+  for (let i = 0, length = openProjects.length; i < length; ++i) {
+    if (sourceFile.indexOf(openProjects[i]) === 0) {
```

Comment:
Should use [`str.startsWith`](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith) instead